### ISSUE TITLE
Some Chromium Apps Crash after setting a theme (perhaps due to unset properties)

### ISSUE THREAD

**Issue by Naixsys** [nwg-piotr/nwg-look#75]
# Basic Info 
Distro:  Fedora 40 Workstation
Kernel: 6.10.10-200.fc40.x86_64 from the official fedora updates repo
WM/DE: Hyprland 0.43.0 from  copr:copr.fedorainfracloud.org:solopasha:hyprland
nwg-look version: 0.2.4 from  copr:copr.fedorainfracloud.org:aeiro:nwg-shell
steam pkg ver: 1.0.0.81 from rpmfusion-non-free-steam

# What Should Happen
Change the theme (Dracula in my case) and not experience application crashing

# What Is Actually Happening 
steamwebhelper and chromium (flatpak) crash. Stack traces indicate issues with libcef and/or libc

# Theory
libcef is having issues parsing certian fields when set by nwg-look. 

# Solution (so far)
Ensure that a theme is set for:
- widget
-  icon
-  cusor 
 
While ensuring that the dir $HOME/.icons does not exist and will not be written to (this can be changed in the `Preferences` section). After doing this, then go to each section and hit `Apply` (not sure if each section needs to be applied individually or if all selected settings are applied once you hit the button. Maybe someone can correct  me on this) 

Then restart your applications.  

# Comments

I am rather certain that this is not the fault of the project but would like to document this issue due to the fact that it caused me heart ache and hope someone can benefit from this. Also, if anyone can guide me as to where I should report this issue, please do. Or even better, if you have any experience reporting issues to either the CEF or Chromium project (do not know where the issue lies), that woudl also be appreciated. 

Thank you

**Comment by creep1ng** [comment]
Thanks for posting your solution! The last week I've started to had some problems related to GTK, I even reinstalled my whole system, and these problem started again after using a theme, and then I see this post with a solution that works... Thanks!

**Comment by Naixsys** [comment]
> Thanks for posting your solution! The last week I've started to had some problems related to GTK, I even reinstalled my whole system, and these problem started again after using a theme, and then I see this post with a solution that works... Thanks!

You're welcome. Glad to have been of help.

**Comment by KyNarec** [comment]
Thanks for this solution, I was fighting steam, brave, thorium, vesktop etc. for the last few days. Resetting GTK Settings fixed it. I'm so glad I can use Linux now, because I had to boot into Windows for the last few days and that was a huge headache for me.

**Comment by hey-adora** [comment]
i also mentioned issue like this on https://github.com/nwg-piotr/nwg-look/issues/74#issuecomment-2386603710
what fixed it for me is `rm ~/.local/share/icons/default/index.theme` if anyone is looking for a fix here.

**Comment by itsmeherefolks** [comment]
> i also mentioned issue like this on [#74 (comment)](https://github.com/nwg-piotr/nwg-look/issues/74#issuecomment-2386603710) what fixed it for me is `rm ~/.local/share/icons/default/index.theme` if anyone is looking for a fix here.

That fixed the problem on Hyprland, too. Thanks!

**Comment by FakeMichau** [comment]
Unsure if I'm seeing the exact same issue but I had steam, discord and polybar crash on me after using nwg-look to set theme on wayland (hyprland) while I usually use x11 (i3) as some settings didn't apply on wayland by default. After doing that I was getting crashes in both wayland and x11.

Looking at a core dump from polybar I noticed it was crashing in libxcb-cursor.so.0. Reapplying just the cursor theme on x11 again fixed the issue for me, although it could've been nwg-look reloading the whole config for all I know.

**Comment by nwg-piotr** [comment]
Guys, I'm not sure. Are you aware of existence of this check button?

![obraz](https://github.com/user-attachments/assets/eed1b459-0082-47c4-bee5-d9053bd87397)

**Comment by nwg-piotr** [comment]
Well, I'll say it one more time: this program applies some gsettings, and __optionally__ exports a couple of ini/config files. No more, no less. There's nothing else under the hood. If you experience issues after changing GTK3 setting, please report them in their appropriate repos.

**Comment by madhur** [comment]
> Well, I'll say it one more time: this program applies some gsettings, and **optionally** exports a couple of ini/config files. No more, no less. There's nothing else under the hood. If you experience issues after changing GTK3 setting, please report them in their appropriate repos.

I agree but it seems to be applying settings in a way which is not a standard practice. You could trigger a warning if such an update is being made warning users of its consequences.

**Comment by nwg-piotr** [comment]
Can you give an example?

**Comment by Morn2** [comment]
Hi all, i had the problem, that gforcenow-electron, discord and steam were crashing after start. I found out that "index.theme" which is stored under /home/Username/.local/share/icons/default was making the crash. When i rename the folder default to default_b, then all the before crashing applications are working. 

The Header of index.theme is: #This file is written by nwg-look. Do not edit. 

This is why i comment to this issue. I hope this is the right plase, else i clould reopen a new issiue with complet details to my setup.
